scripts/deploy.sh

```diff
@@ -122,6 +123,15 @@ if [[ ! -f /etc/openstack_deploy/user_osa_secrets.yml ]] && [[ -f /etc/openstack
   mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
 fi
 
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  cat >> /etc/openstack_deploy/user_osa_secrets.yml <<'EOF'
```

Comment:
This file could also be dropped in tree here:
https://github.com/rcbops/rpc-openstack/tree/mitaka-13.1/rpcd/etc/openstack_deploy
You could name it something like "user_magnum_secrets.yml". It will also get moved over to /etc/openstack_deploy automatically.
These values will also need to be populated with:
```
./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_magnum_secrets.yml
```